src/reformat_libyuv.c

```diff
@@ -431,6 +431,24 @@ avifResult avifImageYUVToRGBLibYUV8bpc(const avifImage * image,
                 return AVIF_RESULT_REFORMAT_FAILED;
             }
             return AVIF_RESULT_OK;
+        } else if (image->yuvFormat == AVIF_PIXEL_FORMAT_YUV422) {
+#if LIBYUV_VERSION >= 1838
+            if (I422ToARGBMatrixFilter(image->yuvPlanes[AVIF_CHAN_Y],
```

Comment:
We should call `I422ToRGB565Matrix()`, right? I am sorry I didn't notice this error earlier.

Also test `AVIF_PIXEL_FORMAT_YUV422` before `AVIF_PIXEL_FORMAT_YUV420`, i.e., in the order of the enum defintion.

Reply:
Sorry it seems like i made a mess with the copy pasting on this CL.

I have updated it now and verified that it works. Please take another look.